.env.example

```diff
@@ -0,0 +1,13 @@
+CDIP_API_ENDPOINT=https://sensors.api.stage.gundiservice.org/v2/observations/
+CDIP_ADMIN_ENDPOINT=https://api.stage.gundiservice.org
+GUNDI_API_BASE_URL=https://api.stage.gundiservice.org
+GCP_PROJECT_ID=cdip-stage-78ca
+KEYCLOAK_AUDIENCE=cdip-admin-portal
+KEYCLOAK_CLIENT_ID=cdip-integrations
+KEYCLOAK_CLIENT_SECRET=REPLACE_FOR_SECRET
+KEYCLOAK_ISSUER=https://cdip-auth.pamdas.org/auth/realms/cdip-dev
+LOOK_BACK_WINDOW_HOURS=720
+LOG_LEVEL=INFO
+METRICS_PROXY_HOST="cdip-metrics-proxy.metrics.svc.cluster.local"
+SENSORS_API_BASE_URL=https://sensors.api.stage.gundiservice.org
+INTEGRATION_LOAD_BATCH_SIZE=100
```

Comment:
this one was removed in integration v2 too

Reply:
Thank you for adding this @davidovichmarcos!
I like the idea of `.env.example` for those willing to define env vars in a `.env` while developing integrations, but looks like some of those env vars were copied from some v1 integration and are not needed here, and others may be missing, so here is an example of the full env vars list needed here:

```
GUNDI_API_BASE_URL
KEYCLOAK_CLIENT_ID
KEYCLOAK_CLIENT_SECRET
KEYCLOAK_ISSUER
LOGGING_LEVEL=DEBUG
REDIS_HOST
REDIS_PORT
REDIS_STATE_DB
SENSORS_API_BASE_URL
```